android-runtime/runtime/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'androidx.core:core-ktx:1.2.0'
+    implementation 'com.squareup.okhttp3:okhttp:4.4.1'
+    api 'com.google.code.gson:gson:2.8.6'
```

Comment:
gson should be "api" and okhttp should be "implementation"

Reply:
I messed it all up. It was already correct before.
okhttp is api because it is exposed on some methods to customize behavior. gson is implementation because it is only used internally